src/magic_enchantment.cpp

```diff
@@ -504,8 +529,10 @@ void enchant_cache::load( const JsonObject &jo, const std::string_view,
                     values_multiply.emplace( value, mult );
                 }
             } catch( ... ) {
-                debugmsg( "A relic attempted to load invalid enchantment %s.  If you updated versions this may be a removed enchantment and will fix itself.",
-                          value_obj.get_string( "value", "" ) );
+                if( legacy_values.find( value_obj.get_string( "value", "" ) ) == legacy_values.end() ) {
+                    debugmsg( "A relic attempted to load invalid enchantment %s.", value_obj.get_string( "value",
+                              "" ) );
```

Comment:
Is removing the second sentence of the message intentional, or an accident?

Reply:
it is intentional, we do not need to say "well maybe it was intentional" if we have a list of enchantments that were intentionally removed

Reply:
This list is incomplete, see e.g. https://github.com/CleverRaven/Cataclysm-DDA/pull/63629/files. For example, `ITEM_ENCUMBRANCE` is missing.

Reply:
It was pre 0.G, we do not support game migrations more than 1 version (ie at this moment, being in the middle of 0.I, only migration from 0.H to 0.I is supported)